stl/inc/exception

```diff
@@ -20,7 +20,9 @@ _STL_DISABLE_CLANG_WARNINGS
 
 _STD_BEGIN
 
+#if _HAS_DEPRECATED_UNCAUGHT_EXCEPTION
 _CXX17_DEPRECATE_UNCAUGHT_EXCEPTION _CRTIMP2_PURE bool __CLRCALL_PURE_OR_CDECL uncaught_exception() noexcept;
+#endif //_HAS_DEPRECATED_UNCAUGHT_EXCEPTION

```

Comment:
Style: We put a space after `//` comments.

Reply:
Yep